packages/Frontend/package.json

```diff
@@ -8,8 +8,6 @@
   "license": "Apache-2.0",
```

Comment:
I removed aws's tools that they made for authentication among other things. Firebase replaces them completely, at least for now.